ArduCopter/mode.h

```diff
@@ -653,7 +662,7 @@ class ModeGuidedNoGPS : public ModeGuided {
     ModeGuidedNoGPS(Copter &copter) :
         Copter::ModeGuided(copter)        { }
 
-    bool init(bool ignore_checks) override;
+    void enter() override;
     void run() override;
 
     bool requires_GPS() const override { return true; }
```

Comment:
Unrelated to this PR, but this is wrong. The mode itself is called NoGPS 🙂

Reply:
Yep - completely bonkers.  See earlier comment!

Reply:
I'm not sure you got my comment here. I wasn't referring to the method name, but the fact that it is returning true in a method that doesn't require GPS.

Reply:
Yep.  We'll add more methods soon, hopefully.